src/fidesops/migrations/versions/55d61eb8ed12_add_default_policies.py

```diff
@@ -331,7 +331,7 @@ def autogenerate_erasure_policy(connection: Connection, client_id: str) -> None:
             "target_id": rule_target_id,
             "target_name": "Fidesops Autogenerated Erasure Target",
             "target_key": "fidesops_autogenerated_erasure_target",
-            "data_category": DataCategory("user.provided.identifiable").value,
+            "data_category": DataCategory("user").value,
```

Comment:
This edits a past data migration.  If any fidesops user has already run this data migration, these changes will not take affect. Instead of editing an old migration, I'd create a new data migration that updates these targets to the new categories.

Reply:
This was unavoidable, the package itself has changed so `user.provided.identifiable` now throws an error and breaks the whole migration

Reply:
Instead of using `DataCategory` then in this migration, I'd update to use strings: "user.provided.identifiable" here, and still add a new migration to update the existing autogenerated rule target.

Reply:
very elegant solution 👍